Description: Suggest installing the packages that are actually available in Ubuntu.
Author: Marc Deslauriers <marc.deslauriers@ubuntu.com>
Forwarded: no, Ubuntu-specific
Updated: 2014-02-17

--- a/virtManager/engine.py
+++ b/virtManager/engine.py
@@ -180,7 +180,7 @@
         # Manager fail message
         msg = _("Could not detect a default hypervisor. Make\n"
                 "sure the appropriate virtualization packages\n"
-                "are installed (kvm, qemu, libvirt, etc.), and\n"
+                "are installed (qemu-system, libvirt-bin), and\n"
                 "that libvirtd is running.\n\n"
                 "A hypervisor connection can be manually\n"
                 "added via File->Add Connection")
